packages/suite/src/support/messages.ts

```diff
@@ -3660,6 +3660,37 @@ export default defineMessages({
         defaultMessage:
             'Legacy uses simpler transaction formats but may result in higher transaction fees and lacks the efficiency and features found in newer address types.',
     },
+    TR_ACCOUNT_TYPE_LEDGER_DESC: {
+        id: 'TR_ACCOUNT_TYPE_LEDGER_DESC',
+        defaultMessage:
+            'Ledger accounts are compatible with Ledger Live derivation paths, enabling smooth  migration from Ledger to Trezor.',
+    },
+    TR_ACCOUNT_TYPE_LEGACY_DESC: {
+        id: 'TR_ACCOUNT_TYPE_LEGACY_DESC',
+        defaultMessage:
+            'Legacy accounts are compatible with Ledger Legacy derivation paths, enabling smooth migration from Ledger to Trezor.',
+    },
+    TR_ACCOUNT_TYPE_NORMAL_EVM_DESC: {
+        id: 'TR_ACCOUNT_TYPE_NORMAL_EVM_DESC',
+        defaultMessage:
+            'The current and most widely accepted method of generating and managing {value} addresses ensures interoperability, security, and support for all types of tokens.',
+        dynamic: true,
```

Comment:
it is not dynamic, because this key is mentioned in the codebase

Reply:
True, thank you